Makefile.toml

```diff
@@ -82,7 +82,7 @@ dependencies = ["test"]
 [tasks.test]
 private = true
 command = "cargo"
-args = ["test", "--all-targets", "--workspace", "--exclude", "website-test"]
+args = ["test", "--all-targets"]
```

Comment:
This change should fix #2403 . As I understand it, tasks are inherited and run for all workspace members anyway, unless overwritten.